datarepo-clienttests/build.gradle

```diff
@@ -93,15 +93,31 @@ test {
 }
 
 task(runTest, dependsOn: 'classes', type: JavaExec) {
-    main = "runner.TestRunner"
+    main = "common.commands.RunTest"
+    classpath = sourceSets.main.runtimeClasspath
+}
+task(lockNamespace, dependsOn: 'classes', type: JavaExec) {
+    main = "common.commands.LockNamespace"
+    classpath = sourceSets.main.runtimeClasspath
+}
+task(unlockNamespace, dependsOn: 'classes', type: JavaExec) {
+    main = "common.commands.UnlockNamespace"
+    classpath = sourceSets.main.runtimeClasspath
+}
+task(lockAndRunTest, dependsOn: 'classes', type: JavaExec) {
```

Comment:
I don't see where you have the dependencies defined for these tasks...do you remove that?

Reply:
I don't think I understand - what dependencies?

Reply:
I thought we wanted `lockAndRunTest` to lock, then run, unlock the environment.  I'm not getting how this making sure those three tasks are running

Reply:
Oh, gotcha! I'm calling the lock and unlock from the main method in the [lockAndRunTest class](https://github.com/DataBiosphere/jade-data-repo/pull/635/files#diff-aef3c0bf149fa0d64faff50b6b95d6d4R18).